notebooks/12.Case_Study-SMS_Spam_Detection.ipynb

```diff
@@ -293,7 +293,7 @@
     "    interesting_coefficients = np.hstack([negative_coefficients, positive_coefficients])\n",
     "    # plot them\n",
     "    plt.figure(figsize=(15, 5))\n",
-    "    colors = [\"red\" if c < 0 else \"blue\" for c in coef[interesting_coefficients]]\n",
+    "    colors = [\"tab:orange\" if c < 0 else \"tab:blue\" for c in coef[interesting_coefficients]]\n",
```

Comment:
I don't know this one... how does that work?

Reply:
Either in this way or using the C# tag.